### ISSUE TITLE
Idea: Rapportive plugin.

### ISSUE THREAD

**Issue by emphaticsunshine** [nylas/N1#2288]
I use [Rapportive](https://rapportive.com/) in my browser to get more information on person who sent email. It's a very useful plugin and would be nice to have support for it in Nylas as well.

**Comment by grinich** [comment]
Nylas N1 already has this. You just need to click the "person" icon in the upper right when viewing a thread.

![image](https://cloud.githubusercontent.com/assets/92417/15435375/af51b3b6-1e70-11e6-9e38-1502fa79edde.png)

**Comment by egsanchez** [comment]
Hi guys!

I'm a big fan of Nylas. I love all the capabilities you have and how fluent it works! But, still missing the Rapportive plugin. Nylas' "Person" icon works only after you sent an email, AFAIK. As @emphaticsunshine I use Rapportive for Gmail to pre-test if the email has a LinkedIn linked.

Is there anything close to this feature and I'm missing it?

Cheers,
